src/components/Editor/DraggableTabs/DroppableTabList.js

```diff
@@ -0,0 +1,24 @@
+import React from "react"
+import { Droppable } from "@hello-pangea/dnd"
+import { TabList } from "react-tabs"
+
+import './DraggableTabs.scss'
+
+const DroppableTabList = ({children, index, ...otherProps}) => {
+  return (
+    <TabList {...otherProps }>
+      <Droppable direction='horizontal' droppableId={`${index}`}>
```

Comment:
do you need to use a template literal for the id here? if so it might be a little clearer (though a touch more verbose) to use `index.toString()`